packages/ui/src/components/va-virtual-scroller/VaVirtualScroller.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div ref="wrapper" class="va-virtual-scroller">
```

Comment:
Do we truly need 3 wrappers?

Reply:
Discussed in Slack.

Yes, we need (but i'm also not happy about this).

First div - wrapper, visible list size. Second - virtual container with size as if all items are rendered. Third - 'sledge', riding on second container in visible zone of the first.